### DISCUSSION TITLE
[tabThumbnailTooltip.uc.js] How to decrease the delay before showing tab thumbnail?

### DISCUSSION THREAD

**Discussion by benzBrake** [aminomancer/uc.css.js#41] [Q&A] [answered]
I want to make it work smooth like vivaldi.

**Comment by aminomancer** [comment, marked as answer]
It should say in the prefs section in the readme. Add a new number pref in about:config, `ui.tooltipDelay` and set it to 100 or 50 or 0. However, the delay can't actually be 0 in practice, because it takes time to generate a thumbnail. It's like taking a screenshot of your screen and resizing it. So the minimum speed is gonna depend on your hardware. But setting the pref as low as you can should reduce the delay since that pref's default value is 300 which is really slow. That's 300 milliseconds so nearly 1/3 of a second
